website/docs/cloud-docs/workspaces/settings/health-assessments.mdx

```diff
@@ -0,0 +1,64 @@
+
+Infrastructure drift means that your real-world infrastructure no longer matches the configuration in your Terraform state file. Drift occurs when a user modifies resources outside of the Terraform workflow. For example, a colleague may update resource configuration directly in the cloud provider console during a production incident, changing the resource attributes from those in tracked in your state file.
+
+> **Hands On:** Try our [Manage Resource Drift](https://learn.hashicorp.com/tutorials/terraform/resource-drift) tutorial on HashiCorp Learn to review how to detect and resolve drift.
+
+Health assessments is a workspace-level setting. When enabled, Terraform Cloud will automatically run a health assessment for that workspace every 24 hours. The assessment checks the actual settings of your infrastructure against those tracked in your workspace's state file.
+
+## Requirements
+
+Workspaces require the following settings to receive health assessments:
+- Terraform version 0.15.4+
+- [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode) or [Agent execution mode](/cloud-docs/agents#configuring-workspaces-to-use-the-agent) for Terraform runs
+
+## Enable Health Assessments
+
+To enable health assessments within a workspace:
+1. Verify that the workspace satisfies the [requirements](#requirements).
+1. Go to the workspace and click **Settings > General**.
+1. Select **Enable** under **Health Assessments**.
+1. Click **Save settings**.
+
+> **Note:** You may also enforce assessments across all workspaces in an organization from the [organization assessments settings](/cloud-docs/users-teams-organizations/organizations/#assessments)
+
+### Health Assessment Scheduling
+
+If there are no active Terraform runs in the workspace, Terraform Cloud starts a health assessment as soon as you enable the feature. If you enable health assessments during a speculative plan, Terraform Cloud will start a health assessment soon after that plan's completion. If you enable health assessments during other types of runs, Terraform Cloud will start the first assessment in about 24 hours.
+
+A health assessment never interrupts or interferes with runs. Terraform Cloud triggers a health assessment if at least 24 hours have passed since the last assessment and there are no active runs in the workspace.
+
+You can start a new run during a health assessment, but it will cancel the current assessment. Terraform Cloud will run the next assessment in 24 hours.
+
+If health assessments are enabled on multiple workspaces, assessments may run concurrently. Health assessments do not affect your concurrency limit. Terraform Cloud also monitors and controls health assessment concurrency, so it will not cause issues for large-scale deployments with thousands of workspaces. However, Terraform Cloud performs health assessments in batches, so health assessments may take longer to complete when you enable it in a large number of workspaces.
```

Comment:
```suggestion
If health assessments are enabled on multiple workspaces, assessments may run concurrently. Health assessments do not affect your concurrency limit. Terraform Cloud also monitors and controls health assessment concurrency, so it will not cause issues for large-scale deployments with thousands of workspaces. However, Terraform Cloud performs health assessments in batches, so health assessments may take longer to complete when you enable them in a large number of workspaces.
```